src/ontology/slurp/omim.tsv

```diff
@@ -0,0 +1 @@
+""
```

Comment:
@matentzn There's nothing in `slurp/` yet, but I was wondering; do you want this dir and the slurp outputs that get put in here to be committed, or should I add this dir to the `.gitignore`?

Reply:
Yes, please commit them.

Reply:
will do!